### DISCUSSION TITLE
Active Directory Auth in Logging Model Events

### DISCUSSION THREAD

**Discussion by itzmj-23** [spatie/laravel-activitylog#1042] [Q&A]
First of all, thanks for the great package.

I am using logging model events, and my web auth is using LDAP (Active Directory). 

In activity log config, all are defaults.

I am having an error of 
`Could not determine a user with identifier CN=Sample Name,OU=ABC,OU=DEF,OU=GHI,OU=JKLM,DC=jklm,DC=com`

Thanks.

**Comment by Gummibeer** [comment]
This error is related to your Auth setup/configuration and doesn't relate to this package.
In case your LDAP provider does special you can use a custom user resolver in v4 of this package.
https://spatie.be/docs/laravel-activitylog/v4/api/causer-resolver
